### ISSUE TITLE
Not working on 18.0.0

### ISSUE THREAD

**Issue by MrSuperdudly1** [hanai3Bi/Switch-OC-Suite#63]
The switch straight up does not boot in 18.0.0

**Comment by ChanseyIsTheBest** [comment]
Long story short Switch-OC-Suite is now EOS and maintained by someone else now.
https://discord.gg/UGPGBQwN
EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel

**Comment by AnvilByte** [comment]
> Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel

anyway to download it without using discord?

**Comment by admiralspeedy** [comment]
> Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel

According to who exactly?

Seems to me that someone just took this project and started making changes to it and only releases it in Discord. That Discord server is absolutely full of people who have absolutely no idea what they're doing, just using AIO bundles without much thought.

**Comment by ChanseyIsTheBest** [comment]
> > Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel
> 
> anyway to download it without using discord?

I'll always keep the latest EOS + sys clk release here in the guide updated
https://rentry.co/SwitchOCSuiteGuide1

EOS 1.1: https://drive.google.com/file/d/1YGIDFTqwoGsmZWmIAvLAgXsu6FjK_Lzi/view?usp=sharing
Updated sys clk: https://drive.google.com/file/d/1fJ7o85W3YMRPJYlwUP_oSQRIIxrv2i5u/view?usp=sharing

**Comment by AnvilByte** [comment]
> > > Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel
> > 
> > 
> > anyway to download it without using discord?
> 
> I'll always keep the latest EOS + sys clk release here in the guide updated https://rentry.co/SwitchOCSuiteGuide1
> 
> EOS 1.1: https://drive.google.com/file/d/1YGIDFTqwoGsmZWmIAvLAgXsu6FjK_Lzi/view?usp=sharing Updated sys clk: https://drive.google.com/file/d/1fJ7o85W3YMRPJYlwUP_oSQRIIxrv2i5u/view?usp=sharing

oh thanks

edit: the guide says to use EPV instead for Erista with the only listed difference being undervolt, is that about it for the differences?

**Comment by ChanseyIsTheBest** [comment]
> > > > Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel
> > > 
> > > 
> > > anyway to download it without using discord?
> > 
> > 
> > I'll always keep the latest EOS + sys clk release here in the guide updated https://rentry.co/SwitchOCSuiteGuide1
> > EOS 1.1: https://drive.google.com/file/d/1YGIDFTqwoGsmZWmIAvLAgXsu6FjK_Lzi/view?usp=sharing Updated sys clk: https://drive.google.com/file/d/1fJ7o85W3YMRPJYlwUP_oSQRIIxrv2i5u/view?usp=sharing
> 
> oh thanks
> 
> edit: the guide says to use EPV instead for Erista with the only listed difference being undervolt, is that about it for the differences?

If you have Erista and you are on Firmware 17 and AMS 1.6.2 use EPV, if you are on Firmware 18 and AMS 1.70 use EOS 1.1.
EPV has CPU Undervolt for Erista which EOS doesn't have.

**Comment by OMompm** [comment]
@ChanseyIsTheBest 
Hi, I checked out the Discord and understandably most of the effort is focused on Mariko.
I have an Erista, only play docked and only care about cleaning up FPS drops from whatever the game runs by default. And maybe some light graphics mods. 
Even if future Erista-focused EVPs don't get made, can I just get EOS and use it without UV? 
As I understand it, UV can give you better temps and lower energy draw.

**Comment by ChanseyIsTheBest** [comment]
> @ChanseyIsTheBest Hi, I checked out the Discord and understandably most of the effort is focused on Mariko. I have an Erista, only play docked and only care about cleaning up FPS drops from whatever the game runs by default. And maybe some light graphics mods. Even if future Erista-focused EVPs don't get made, can I just get EOS and use it without UV? As I understand it, UV can give you better temps and lower energy draw.

Yeah you can get EOS and use it with UV. It's just missing CPU UV for Erista you can still use GPU UV on Erista (the more important one) anyway

**Comment by AnvilByte** [comment]
> > > > > Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel
> > > > 
> > > > 
> > > > anyway to download it without using discord?
> > > 
> > > 
> > > I'll always keep the latest EOS + sys clk release here in the guide updated https://rentry.co/SwitchOCSuiteGuide1
> > > EOS 1.1: https://drive.google.com/file/d/1YGIDFTqwoGsmZWmIAvLAgXsu6FjK_Lzi/view?usp=sharing Updated sys clk: https://drive.google.com/file/d/1fJ7o85W3YMRPJYlwUP_oSQRIIxrv2i5u/view?usp=sharing
> > 
> > 
> > oh thanks
> > edit: the guide says to use EPV instead for Erista with the only listed difference being undervolt, is that about it for the differences?
> 
> If you have Erista and you are on Firmware 17 and AMS 1.6.2 use EPV, if you are on Firmware 18 and AMS 1.70 use EOS 1.1. EPV has CPU Undervolt for Erista which EOS doesn't have.

my EmuMMC is on 17.0.0 and AMS 1.7, I used EPV and it worked fine, also the guide didn't say to use EOS for AMS 1.7 on Erista afaik

**Comment by AnvilByte** [comment]
> @ChanseyIsTheBest Hi, I checked out the Discord and understandably most of the effort is focused on Mariko. I have an Erista, only play docked and only care about cleaning up FPS drops from whatever the game runs by default. And maybe some light graphics mods. Even if future Erista-focused EVPs don't get made, can I just get EOS and use it without UV? As I understand it, UV can give you better temps and lower energy draw.

yeah I have an Erista too, I got lucky and was able to buy a "broken" unpatched unit that only needed a pretty easy fix for pretty cheap, getting a modded Mariko would be harder unless I buy it pre-modded, but iirc hardmodded switches are kinda illegal to sell in Canada though

**Comment by MasqueradeX45** [comment]
> Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel

Do you have an updated discord invite link?

**Comment by ChanseyIsTheBest** [comment]
> > Long story short Switch-OC-Suite is now EOS and maintained by someone else now. https://discord.gg/UGPGBQwN EOS 1.1 prerelease works with AMS 1.7.0 and Firmware 18 and is in pins on the #overclocking channel
> 
> Do you have an updated discord invite link?

https://discord.gg/VndKxFg7EE

This link should always work I think

**Comment by RalphJB12** [comment]
Any updates on this for 18.1.0 and Atmosphere 1.7.1?

**Comment by pgalonza** [comment]
I tried to build it myself. It works for me.

https://github.com/pgalonza/ns-Atmosphere/tree/hos_18
